sample/.buckconfig

```diff
@@ -0,0 +1,6 @@
+
+[android]
```

Comment:
if we use `Gradle` for Android, we don't need `Buck`

Reply:
This came up with the react-native init project

Reply:
yes I know, just afraid to make the sample heavier and even harder to set up

Reply:
I'm afraid of spending more hours "refactoring" this. I'd like to get this working first. We can remove all unecessary code after. I'd focus on adding cocoa 5.0 before that.